src/Robo/Plugin/Commands/ValidateConfigCommands.php

```diff
@@ -50,7 +50,7 @@ public function validateDrupalConfig($siteDir = 'default'): void
             ->run();
         $drushOutput = trim($result->getOutputData());
         $configJson = json_decode($drushOutput);
-        if (!is_null($configJson)) {
```

Comment:
...and an empty string is not `null`, making this throwing the "We're-All-Gonna-Die-WTFBBQ!!!!!" `TaskException` when in fact all is well.